### PR TITLE
Fix platform invoke data types

### PR DESCRIPTION
**Pull request by nxtn**
Contributes to #11396

*wtypes.h* is for Win16 only. These types are defined in either *winnt.h* or *minwindef.h*.

### CODE REVIEW

File: docs/framework/interop/marshaling-data-with-platform-invoke.md [hunk 1]
```diff
@@ -19,35 +19,35 @@ To call functions exported from an unmanaged library, a .NET Framework applicati
...
+|`CHAR`|`char`|<xref:System.Char?displayProperty=nameWithType>|Decorate with ANSI.|
+|`WCHAR`|`wchar_t`|<xref:System.Char?displayProperty=nameWithType>|Decorate with Unicode.|
+|`LPSTR`|`char *`|<xref:System.String?displayProperty=nameWithType> or <xref:System.Text.StringBuilder?displayProperty=nameWithType>|Decorate with ANSI.|
+|`LPCSTR`|`const char *`|<xref:System.String?displayProperty=nameWithType> or <xref:System.Text.StringBuilder?displayProperty=nameWithType>|Decorate with ANSI.|
+|`LPWSTR`|`wchar_t *`|<xref:System.String?displayProperty=nameWithType> or <xref:System.Text.StringBuilder?displayProperty=nameWithType>|Decorate with Unicode.|
+|`LPCWSTR`|`const wchar_t *`|<xref:System.String?displayProperty=nameWithType> or <xref:System.Text.StringBuilder?displayProperty=nameWithType>|Decorate with Unicode.|
+|`FLOAT`|`float`|<xref:System.Single?displayProperty=nameWithType>|32 bits|
+|`DOUBLE`|`double`|<xref:System.Double?displayProperty=nameWithType>|64 bits|
 
 For corresponding types in [!INCLUDE[vbprvblong](../../../includes/vbprvblong-md.md)], C#, and C++, see the [Introduction to the .NET Framework Class Library](../../../docs/standard/class-library-overview.md).
```
Comment (rpetrusha) [inline docs/framework/interop/marshaling-data-with-platform-invoke.md]:
```suggestion
For corresponding types in Visual Basic, C#, and C++, see the [Introduction to the .NET Framework Class Library](../../standard/class-library-overview.md#system-namespace).
```

File: docs/framework/interop/marshaling-data-with-platform-invoke.md [hunk 2]
```diff
@@ -19,35 +19,35 @@ To call functions exported from an unmanaged library, a .NET Framework applicati
...
 
 This section provides samples that demonstrate how to create managed function prototypes for passing arguments to and receiving return values from functions exported by unmanaged libraries. The samples also demonstrate when to use the <xref:System.Runtime.InteropServices.MarshalAsAttribute> attribute and the <xref:System.Runtime.InteropServices.Marshal> class to explicitly marshal data.
 
 ## Platform invoke data types
 
 The following table lists data types used in the Win32 API (listed in Wtypes.h) and C-style functions. Many unmanaged libraries contain functions that pass these data types as parameters and return values. The third column lists the corresponding .NET Framework built-in value type or class that you use in managed code. In some cases, you can substitute a type of the same size for the type listed in the table.
 
-|Unmanaged type in Wtypes.h|Unmanaged C language type|Managed type name|Description|
+|Unmanaged type in Win32 APIs|Unmanaged C language type|Managed type|Description|
```
Comment (rpetrusha) [inline docs/framework/interop/marshaling-data-with-platform-invoke.md]: Perhaps Windows APIs instead of Win32 APIs?

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @NextTurn. I've left two suggestions and a comment.

File: docs/framework/interop/marshaling-data-with-platform-invoke.md (hunk 2 above)
Comment (NextTurn) [reply docs/framework/interop/marshaling-data-with-platform-invoke.md]: I agree.

> Note that this was formerly called the Win32 API. The name Windows API more accurately reflects its roots in 16-bit Windows and its support on 64-bit Windows.

https://docs.microsoft.com/windows/desktop/apiindex/windows-api-list

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @NextTurn, for the additional changes. I'll merge your PR now.

**Pull request merged**